src/coreComponents/physicsSolvers/solidMechanics/kernels/ImplicitSmallStrainQuasiStatic_impl.hpp

```diff
@@ -128,6 +129,15 @@ void ImplicitSmallStrainQuasiStatic< SUBREGION_TYPE, CONSTITUTIVE_TYPE, FE_TYPE
 
   FE_TYPE::symmetricGradient( dNdX, stack.uhat_local, strainInc );
 
+  for( int is = 0; is < 6; ++is )
+  {
+    if( q == 0 )
+    {
+      m_incStrain[k][is] = 0.0;
+    }
+    m_incStrain[k][is] += strainInc[is]*detJxW/m_elementVolume[k];
```

Comment:
same code seems to be repeated a few times.

Reply:
I agree - Is there a place you would recommend placing this function? Its seems like a lot of the quadrature point kernel is kind of repeated in the various places it is used.